Unix/http/httpclient.c

```diff
@@ -813,6 +817,37 @@ static Http_CallbackResult _ReadData(
         _WriteTraceFile(ID_HTTPCLIENTRECVTRACEFILE, (char*)(handler->recvPage + 1), handler->receivedSize);
     }
 
+
+    if (!handler->ssl && handler->encrypting)
+    {
+
+#if ENCRYPT_DECRYPT
```

Comment:
I would expect that if we decided to deactivate encrypt we would either put the same ifdef around the  selection of "--encryption http"  so that it was not possible to select it from the command line or from HttpClient_NEwConnector2